taskfile/node_http.go

```diff
@@ -60,6 +56,11 @@ func (node *HTTPNode) Remote() bool {
 }
 
 func (node *HTTPNode) Read(ctx context.Context) ([]byte, error) {
+	url, err := RemoteExists(ctx, node.logger, node.URL, node.timeout)
```

Comment:
You removed the context.WithTimeout

You could put it back here before this

Because the timeout should be used with the head and the get


```suggestion
  ctx, cancel := context.WithTimeout(ctx, node.timeout)
  defer cancel()
	url, err := RemoteExists(ctx, node.logger, node.URL)
```

You are not using the timeout right now in remote except for the error

Reply:
The timeout is set in the reader : 

```golang
ctx, cf := context.WithTimeout(context.Background(), r.timeout)
defer cf()

// Read the file
b, err = node.Read(ctx)
```

Reply:
Then you should catch the CTX error in the caller ? This way, you don't introduce the timeout parameter, that is only used for reporting errors,no?

Maybe I missed something again 😅😁